docker-compose.devel.yml

```diff
@@ -52,6 +52,8 @@ services:
     build:
       context: fabric8-analytics-data-model/
       dockerfile: Dockerfile.data-model
+    volumes:
+       - ./fabric8-analytics-data-model/src:/src:z
```

Comment:
Can we mount it as `ro,z` ?

Reply:
Sure, that should work. Let me update it.

Reply:
Updated!